docs/reference/rockcraft.yaml.rst

```diff
@@ -78,10 +78,14 @@ special value ``bare`` means that the ROCK will have no base system at all,
 which is typically used with static binaries or
 :ref:`Chisel slices <chisel_explanation>`.
 
+.. note::
```

Comment:
Looks like this:
![image](https://github.com/canonical/rockcraft/assets/1951299/36991b36-381f-4e49-b9d7-5febefb50cb9)